Controller/ContentTypeController.php

```diff
@@ -127,6 +140,8 @@ public function viewContentTypeGroupAction($contentTypeGroupId)
             )->createView();
 
             $canDeleteById[$contentTypeId] = $canDelete && !$this->contentTypeService->isContentTypeUsed($contentType);
+
+            $languageById[$contentTypeId] = $this->getVerifiedContentTypeLanguageCode($contentType);
```

Comment:
@andrerom Right, we should only load the languages once. But we can't just choose a language at random, so we need the list of prioritised languages, and we also need the main CT language in case none of the prioritised ones exist in the CT. I can rework the helpers to avoid multiple loading.